client/src/pages/locations/Form.js

```diff
@@ -84,6 +84,21 @@ const LocationForm = ({ edit, title, initialValues }) => {
     }
   }
 
+  // Location types to be shown to admins in the new location page.
+  const locationTypesAdmin = [
+    Location.LOCATION_TYPES.ADVISOR_LOCATION,
+    Location.LOCATION_TYPES.PRINCIPAL_LOCATION,
+    Location.LOCATION_TYPES.PINPOINT_LOCATION,
+    Location.LOCATION_TYPES.GEOGRAPHICAL_AREA,
+    Location.LOCATION_TYPES.VIRTUAL_LOCATION
+  ]
+
+  // Location types to be shown to super users in the new location page.
+  const locationTypesSuperUser = [
```

Comment:
This can be moved outside the components and the name can be changed to LOCATION_TYPES_SUPER_USER.